spring-boot-project/spring-boot-cli/src/main/java/org/springframework/boot/cli/app/SpringApplicationLauncher.java

```diff
@@ -69,14 +70,11 @@ public Object launch(Class<?>[] sources, String[] args) throws Exception {
 	}
 
 	private String getSpringApplicationClassName() {
-		String className = System.getProperty("spring.application.class.name");
-		if (className == null) {
-			className = getEnvironmentVariable("SPRING_APPLICATION_CLASS_NAME");
-		}
-		if (className == null) {
-			className = DEFAULT_SPRING_APPLICATION_CLASS;
-		}
-		return className;
+		return Objects.requireNonNullElse(amount, 
```

Comment:
I am not a maintainer so feel free to ignore my comment.

It's questionable if using `Objects.requireNonNullElse` improves anything here - personally I don't find it more readable, rather the opposite.

But even bigger problem is that the changed code does not compile (what is `amount`?) and does not do what the original code did.

Reply:
I agree. I actually find this harder to read.